primefaces/src/main/java/org/primefaces/component/imagecropper/ImageCropperRenderer.java

```diff
@@ -284,50 +228,38 @@ else if (stream != null) {
      * returns the value obtained by ResourceHandler.createResource().
      */
     private Resource getImageResource(FacesContext facesContext, ImageCropper imageCropper) {
```

Comment:
i wonder in general if we need this method?
we could just evaluate the EL, without splitting and so on?

Reply:
TBH i would revert that
#{resource[test:test]} must just work, is specified by JSF and resolve internally to the correct portlet

Reply:
To discuss in another ticket then, here, the idea is just to clean up the code